zeppelin-web/package.json

```diff
@@ -40,50 +38,53 @@
     "angular-sanitize": "1.5.7",
     "angular-touch": "1.5.7",
     "angular-ui-ace": "0.2.3",
-    "angular-websocket": "1.0.14",
+    "angular-ui-bootstrap": "~2.5.0",
+    "angular-ui-grid": "4.4.6",
+    "angular-websocket": "1.1.0",
     "angular-xeditable": "0.1.12",
+    "ansi_up": "^2.0.2",
+    "ap-viewport-watch": "0.1.43",
     "bootstrap": "~3.4.1",
     "bootstrap3-dialog": "~1.35.4",
     "clipboard": "~1.5.5",
     "d3": "^3.4.4",
+    "diff": "3.3.0",
+    "diff-match-patch": "1.0.0",
     "es5-shim": "~3.1.0",
     "excel-builder": "2.0.0",
+    "github-markdown-css": "2.6.0",
+    "grunt-angular-templates": "^0.5.7",
+    "grunt-dom-munger": "^3.4.0",
+    "headroom.js": "^0.9.3",
     "highlight.js": "9.4.0",
     "jquery": ">=1.8",
     "jquery-ui": ">=1.8",
     "jquery.scrollto": "~2.1.2",
-    "diff": "3.3.0",
     "json3": "~3.3.1",
     "jszip": "2.6.1",
     "lodash": "~3.9.3",
+    "mathjax": "3.0.0",
```

Comment:
I thought mathjax should not be updated to 3.0.0. See https://github.com/apache/zeppelin/pull/4786

Reply:
You're right.

Check the followings.
I think there was a mistake during rebase.

https://github.com/Henry-Hong/zeppelin/blob/e2b12d1a5a53f17c65a87ac9997b294aeff946b0/zeppelin-web/package.json#L66